make.config.in

```diff
@@ -48,16 +48,16 @@ MKDIR	= @MKDIR_P@
 ####################################################################
 
 LD		= $(CXX)
-LDFLAGS = @LDFLAGS@
+LDFLAGS = $(LDFLAGS) @LDFLAGS@
 
 ####################################################################
 # Extra (optional) source files, includes and libs
 # NOTE: EXTRA_SRC must include a solver (solver.cxx or ida_solver.cxx)
 #       and a file format (nc_format.cxx or pdb_format.cxx)
 ####################################################################
 
-EXTRA_INCS	  = @EXTRA_INCS@
-EXTRA_LIBS	  = @EXTRA_LIBS@ @OPENMP_CXXFLAGS@
+EXTRA_INCS	  = $(CPPFLAGS) @CPPFLAGS@ @EXTRA_INCS@
+EXTRA_LIBS	  = $(LIBS) @LIBS@ @EXTRA_LIBS@ @OPENMP_CXXFLAGS@
```

Comment:
Not 100% this is the right thing to do, but a) we do it for `CXXFLAGS` above, and b) it seems to work